libs/protocol-client/src/lib/client/types.ts

```diff
@@ -0,0 +1,5 @@
+import { GetUserCustomQuery, ListContributionsQuery } from '../protocol-types';
+
+export type ContributionItem = ListContributionsQuery['result'][number];
```

Comment:
What is the number meant to do here

Reply:
`ListContributionsQuery['result']` returns an array, and we are interested in the element type of that array. 

`T[number]` refer to the element type of the array as `T`.

Reply:
Good to know

Reply:
We will move these type to a separate library called ui-types. Which will hold aliases for client types that will be used in frontend libraries. @jonathanprozzi Does prefixing alias types with UI e.g. UIContribution work for you? The main benefit is that our editors can easily find the type, and devs will have a better dev experience (better readibility, discoverability, etc..).

Reply:
That sounds good to me! We have a types folder in our frontend and I put a “forms.ts” that has all the form related types in it. We can then prefix those as well @alexkeating @amrro

Reply:
I have these in the other PR. I can update these. So for example we’d do something like UIContributionFormValues instead of ContributionFormValues since those are UI types